package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/mapbox-gl-geocoder",
-  "version": "4.5.1",
+  "version": "4.5.2",
```

Comment:
lets omit this this PR since it will be done as part of the release per https://github.com/mapbox/mapbox-gl-geocoder/blob/master/CONTRIBUTING.md#deploying

Reply:
aha makes sense, thanks. Will remove from here and package-lock.json. 

Does this line need to be removed too until the release is cut?
https://github.com/mapbox/mapbox-gl-geocoder/pull/350/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3

Reply:
yes, you can leave it under the master header until the release is cut and then the entry can be made.